README.rst

```diff
@@ -19,7 +19,26 @@ Usage
 
 Unlike ``{% extends %}``, the ``{% embed %}`` tag can be repeated in the template and does not have to occur as the first tag (essentially like the ``{% include %}`` tag). Likewise, ``{% embed %}`` inherits the context by default, and allows you to pass additional context using the ``with`` keyword. Passing only the explicitly listed variables can be achieved using the ``only`` keyword.
 
-*TODO add an example*
+
+
+``template.html``:
+
+::
+
+    <div class="warp">
+        Var : {{ var_name|default:"default_value" }}
+        {% block template_content %}{% endblock %}
+    </div>
+
+``render.html``:
+
+::
+
+    {% embed 'template.html' with var_name="var_value" %}
```

Comment:
While this demonstrates the basic usage, I was looking for a slightly more elaborate example that also presents a convincing use-case (#2). This example does not distinguish it from `{% extends %}`.. I'll think about this one for a bit.